client/components/card/README.md

```diff
@@ -32,7 +32,8 @@ Name | Type | Default | Description
 --- | --- | --- | ---
 `className` | `string` | null | Adds CSS classes.
 `href` | `string` | null | URL of the card. Adds a right chevron icon.
-`tagName` | `string` | null | Allows you to control the tag name of the card wrapper (only if `href` is not specified).
+`fakeLink` | `bool` | false | Used for cards without a `href` that need to display as a link. The card will render as an accessible button with the same visual appearance as a link (including the right chevron icon).
```

Comment:
I wasn't sure about adding this `fakeLink` option, but it seemed like the best way to make the "Renew Now" button (which is a button since it needs to run code on click) look like the other links in the nav.

There is some precedent for other code in Calypso partially trying to make button cards look like links and doing it via custom CSS (e.g. the "Remove" nav item on subscriptions that can be removed).  But in that case the goal wasn't to make it look exactly the same as the other links, whereas in this case I want it to.  So I figured it was best to add this functionality to the `Card` component.